coderd/rbac/object.go

```diff
@@ -22,6 +22,12 @@ var (
 		Type: "workspace",
 	}
 
+	// ResourceWorkspace Create on behalf of another user.  Org owner.
+	// create - create workspaces
+	ResourceWorkspaceCreationForUser = Object{
```

Comment:
Doesn't seem like this is used anywhere. I'm not sure how this gets rolled into the permission request, but @Emyrk would know!

Reply:
@Emyrk I copied what we did with the [Audit Log permission](https://github.com/coder/coder/pull/3464/commits/b7f70ba39b7c2190a2d039b6a3eb7167c0b3c3ea#diff-fe38937b4f09e6088e82d24a26b88c50896942beff93cf92be136db9e3369a29R27) we paired on. I'm basically trying to add a new permission to the FE (`createWorkspaceForUser`). This permission should be granted to owners, only. Does this look right to you or nah?

Reply:
This is incorrect. You are not making a new resource type, we are just checking "can we make a new workspace in this org for this user". So the object is `ResourceWorkspace.InOrg(organizationID).WithOwner(workspaceOwner)` and the action is `ActionCreate`

Reply:
So we should remove this new resource type.

Reply:
See https://github.com/coder/coder/pull/4247/files/ce46918669b14ec2a1196981ccf1702e38a4bc14#r983662962